core/setup.py

```diff
@@ -70,9 +70,9 @@ def read(fname):
         'json-rpc>=1.12,<2',
         'werkzeug>=0.15,<0.17',
         'dataclasses==0.6;python_version<"3.7"',
-        'hologram==0.0.12',
         'logbook>=1.5,<1.6',
         'typing-extensions>=3.7.4,<3.8',
+        'jsonschema',
```

Comment:
- Woah... are we done with `hologram` entirely??
- We'll need `dbt-mashumaro` here somewhere. I think [`dependency_links`](https://python-packaging.readthedocs.io/en/latest/dependencies.html#packages-not-on-pypi)?

Reply:
Have removed the Hologram code in dbt.dataclass_schema and updated Hologram.